tutorials/60-minute-blitz.jl

```diff
@@ -273,9 +273,9 @@ Flux.train!(loss, params(m), [(data,labels)], opt)
 # It also has a number of dataloaders that come in handy to load datasets.
 
 using Statistics
-#using CuArrays
-using Zygote
+using Flux.Zygote
 using Flux, Flux.Optimise
+# using Flux.CuArrays
```

Comment:
I guess we can just remove these lines entirely, since CuArrays is always loaded anyway.